olmo/config.py

```diff
@@ -49,6 +49,8 @@
     "WandbConfig",
     "CompilerConfig",
     "WandbConfig",
+    "DistributedStrategy",
+    "DDPGradSyncMode",
```

Comment:
Missing `DDPConfig` I believe:
```suggestion
    "DDPGradSyncMode",
    "DDPConfig",
```